src/components/mediaDocumentExternal/formatted-type.ts

```diff
@@ -0,0 +1,17 @@
+import { MediaImage } from '@/components/mediaDocument/formatted-type'
+
+export interface MediaDocumentExternal {
+  id: string
+  name: string
+  drupal_mid: number | string
+  drupal_vid: number | string
+  description: string | null
+  mimeType: string
+  url: string
+  fileName: string
+  /* TODO: `thumbnail` field exists but is unused in template (neither drupal nor next CLP->ResourcesPanel.tsx)
+   * The data matches DrupalMediaImage['image'] but with type 'file--file'. This deserves more consideration
+   * on how to handle it, but doing so will not serve the scope of the following ticket:
+   * https://github.com/department-of-veterans-affairs/va.gov-cms/issues/22445
+   */
```

Comment:
Looks like it's used in campaign landing page's promos and stories sections, for what it's worth.

From `content-build`:
<img width="2091" height="734" alt="image" src="https://github.com/user-attachments/assets/863349f8-98f8-40e1-97bf-080dc0c720c3" />